modules/tinymce/src/core/main/ts/api/util/XHR.ts

```diff
@@ -15,7 +15,7 @@ export interface XHRSettings {
   async?: boolean;
   content_type?: string;
   crossDomain?: boolean;
-  data?: string;
+  data?: any;
```

Comment:
We found this was incorrect, as it allows `FormData`, `Blob`, etc... as well. `any` isn't quite correct though, so I'll fix this up tomorrow.

Reply:
Fixed